bin/install/toran.sh

```diff
@@ -62,11 +63,14 @@ else
 fi
 
 # Create directory mirrors
-if [ ! -e $DATA_DIRECTORY/mirrors ]; then
+if [ ! -d $DATA_DIRECTORY/mirrors ]; then
     echo "Creating mirrors directories..."
     mkdir -p $DATA_DIRECTORY/mirrors
 fi
-ln -sfn $DATA_DIRECTORY/mirrors $WORK_DIRECTORY/web/mirrors
+if [ -d $WORK_DIRECTORY/web/mirrors ]; then
+    rm -rf $WORK_DIRECTORY/web/mirrors
```

Comment:
I think that composer need presence of this folder (symbolic link) in the same path as the folder repo